### PR TITLE
[DECLUTTER] remove PGP verification on windows

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This ready for review branch removes the tutorial about pgp verification on windows, as this is not necessary because the installer is signed and verified automatically.

I am currently missing a screenshot of how the GUI signature verification looks, can anyone please send me one?

### CODE REVIEW

**Comment by MaxHillebrand** [comment]
I am unsure why these merge conflicts exist, can anyone help out?

File: docs/using-wasabi/InstallPackage.md [hunk 1]
```diff
@@ -28,80 +28,20 @@ If you have personally verified zkSNACKs' PGP public key [${zksnacksPublicKeyFin
...
-
-![](/InstallWindowsKleopatraValidSig.png)
-
-:::tip
-The output from the verify command may contain `WARNING: This key is not certified with a trusted signature!`.
-You can ignore this, but if you want to fully verify your download, you need to ask people you trust to confirm that the key fingerprint belongs to zkSNACKs.
-:::
-
-9. You can install Wasabi by double clicking the `.msi` and following the GUI instructions.
+3. Make sure that you see this window that verifies the installer was signed by zkSNACKs ltd.
```
Comment (molnard) [inline docs/using-wasabi/InstallPackage.md]:
```suggestion
3. Make sure that you see this window that verifies the installer was signed by zkSNACKs Limited.
```

**Comment by yahiheb** [comment]
I don't understand why did we remove the PGP verification on Windows?

**Comment by molnard** [comment]
You can still do it. But it is mentioned in detail. There is code signing on the MSI file. Windows will verify the content, extract and check the certificate automatically. The user only has to verify that the signature comes from zkSNACKs Limited.

**Comment by yahiheb** [comment]
I agree that it should be simple.
However we could have moved the manual PGP verification on Windows in an advanced section so users don't have to trust Windows.

**Comment by molnard** [comment]
Yes I agree.

**Pull request merged**